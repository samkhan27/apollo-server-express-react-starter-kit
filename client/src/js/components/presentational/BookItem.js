import React from 'react';

const Book = ({
  title,
  author
}) => (
  <div>
    <p>
      {title}: {author}
    </p>
  </div>
);

export default Book;