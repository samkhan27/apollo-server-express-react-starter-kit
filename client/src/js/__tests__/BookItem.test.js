import React from 'react';
import BookItem from '../components/presentational/BookItem';
import { create, act } from 'react-test-renderer';

test('It loads the book item component with title and author', () => {
  let component;
  act(() => {
    component = create(
      <BookItem title="Game of Thrones" author="George RR Martin" />
    );
  });

  expect(component.toJSON()).toMatchSnapshot();
});