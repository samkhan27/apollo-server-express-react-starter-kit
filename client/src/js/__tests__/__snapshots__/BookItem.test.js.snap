// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It loads the book item component with title and author 1`] = `
<div>
  <p>
    Game of Thrones
    : 
    George RR Martin
  </p>
</div>
`;
